py/torch_tensorrt/dynamo/_exporter.py

```diff
@@ -379,7 +366,11 @@ def inline_trt_modules(
         trt_module_node = trt_module_node[0]
         assert trt_module_node.args
 
-        num_outputs = len(outputs_map[trt_module_node.name])
+        if "val" not in trt_module_node.meta:
+            raise ValueError(
+                f"trt_module_node: {trt_module_node.name} does not have the meta['val'] info, it should be set during dynamo compile_module step."
```

Comment:
Consider using "trt_module_node: {trt_module_node.name} does not have the metadata which should be set during compile_module step"

Reply:
modified as suggested